VideoWeb/VideoWeb/ClientApp/src/app/services/video-filter.service.ts

```diff
@@ -34,7 +38,8 @@ export class VideoFilterService {
     activeFilter: BackgroundFilter;
     imgs: Map<BackgroundFilter, HTMLImageElement> = new Map();
 
-    constructor(private logger: Logger) {
+    constructor(private logger: Logger, private configService: ConfigService, private deviceTypeService: DeviceTypeService) {
+        this.configService.getClientSettings().subscribe(settings => (this.enableVideoFilters = settings.enable_video_filters));
```

Comment:
If this never changes during use of the app could you put a pipe(first()) before the subscribe.

Reply:
We could, but that's somewhat taken care of by the service itself

Reply:
Fair enough it's not needed for http calls since they complete themselves but I'm not clear how the config service works with the caching. I think it would be good to be explicit here.